schema/v1.1/ast/metrics.go

```diff
@@ -0,0 +1,52 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ast // import "go.opentelemetry.io/otel/schema/v1.1/ast"
+
+import (
+	ast10 "go.opentelemetry.io/otel/schema/v1.0/ast"
+	types10 "go.opentelemetry.io/otel/schema/v1.0/types"
+	types11 "go.opentelemetry.io/otel/schema/v1.1/types"
+)
+
+// Metrics corresponds to a section representing a list of changes that happened
+// to metrics schema in a particular version.
+type Metrics struct {
+	Changes []MetricsChange
+}
+
+// MetricsChange corresponds to a section representing metrics change.
+type MetricsChange struct {
+	RenameMetrics    map[types10.MetricName]types10.MetricName `yaml:"rename_metrics"`
```

Comment:
Same for types, we use types from 1.0 package. I can copy/paste them or type alias instead.

Reply:
Alias should be good :+1: